### DISCUSSION TITLE
Multi file/directory select with CTRL Button

### DISCUSSION THREAD

**Discussion by thesabrah** [spacedriveapp/spacedrive#2286] [Feature Request]
I know this feature exists (the multi-select is done in cursor only ) but when I press  `CTRL` button don't work in multi-select the file/dir

**Comment by utkubakir** [comment]
We use `Shift`, which OS are you on and why `CTRL`?

**Reply by thesabrah** [reply]
I use Windows and I always use `CTRL` to select not `SHIFT`, generally thanks to the info

**Reply by utkubakir** [reply]
Oh okay, we will change it to `CTRL` for windows on the next update

**Comment by niikeec** [comment]
The selection behavior will lean towards a more native experience with [this PR](https://github.com/spacedriveapp/spacedrive/pull/2367), which will be available in the next update.
